### ISSUE TITLE
Fix problem with vignette

### ISSUE THREAD

**Issue by joshbrowning2358** [SWS-Methodology/faoswsImputation#2]
The current vignette depends on the faoswsProduction package (which contains datasets and a good example of how to use faoswsImputation).  However, faoswsProduction depends on faoswsImputation and thus we have a circular dependency if we build the faoswsImputation vignette.  Thus, the vignette has simply been ignored in the past, and this needs to be fixed.

**Comment by joshbrowning2358** [comment]
I've fixed this issue.  We can close this issue and return to building vignettes for faoswsImputation.

**Comment by sebastian-c** [comment]
I'm still getting the following when building the vignette:

```
Quitting from lines 734-738 (faoswsImputation.Rnw) 
Error: processing vignette 'faoswsImputation.Rnw' failed with diagnostics:
Type of RHS ('double') must match LHS ('integer'). To check and coerce would impact performance too much for the fastest cases. Either change the type of the target column, or coerce the RHS of := yourself (e.g. by using 1L instead of 1)
Execution halted
Error: Command failed (1)
Execution halted
```

It seems to be in this block:

```
<<fig.height=3>>=
model = ensembleModel(model = defaultMars, extrapolationRange = Inf,
                      level = "local")
imputationParameters$ensembleModels = list(model)
imputeVariable(data = exampleData, imputationParameters = imputationParameters)
@
```

I'm guessing it's a type problem with a data.table merge. Does it happen on your machine as well?

**Comment by joshbrowning2358** [comment]
Strange...  I have seen that error before with this vignette, but it seems to be running fine on my machine.  Can you try it again after a git pull?  I pushed a few changes shortly after closing this issue...

**Comment by sebastian-c** [comment]
My latest commit hash is a41c1fff725ff48bae5922c71598a24b46ab26c5, which is where the master branch is now. I'm getting some inconsistent errors, though. 

I tried knitting the Rnw file myself using Rstudio, but it breaks really early:

```
Error in match.arg(options$results, c("verbatim", "tex", "hide")) : 
  'arg' should be one of "verbatim", "tex", "hide"
Calls: <Anonymous> -> SweaveParseOptions -> check -> match.arg
Execution halted
```

I tried using `purl` to grab out the code, but then I get stuck on line 19 (`okra = fread(input = file)`) because `system.file` doesn't find an extdata directory. Using the `knit` function directly on the document gives the same results.

So frankly, I can't work out how it gets as far as it does in the first place when knitting the vignette.
